.github/workflows/test-sdk-environment.yml

```diff
@@ -0,0 +1,69 @@
+name: Test SDK with Local Development
```

Comment:
What is this thing? We don't have tests in this repo.
Did you want this file to be in `agents-ui` repo perhaps?